WooCommerce build from source fails due to missing `puppeteer-utils` dependency (?)

Note this is a dev-tooling bug that's been bugging me - not sure if there's something weird about my setup.

**Describe the bug**
When I build WooCommerce core from source, it fails with this error:

```

Running "eslint:src" (eslint) task
Warning: Cannot read config file: /Users/rua/_automattic/ventures-repos/woocommerce/.eslintrc.js
Error: Cannot find module '@automattic/puppeteer-utils'
Require stack:
- /Users/rua/_automattic/ventures-repos/woocommerce/tests/e2e/env/config/jest.config.js
- /Users/rua/_automattic/ventures-repos/woocommerce/tests/e2e/env/config/use-config.js
- /Users/rua/_automattic/ventures-repos/woocommerce/.eslintrc.js
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/gruntify-eslint/node_modules/eslint/lib/config/config-file.js
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/gruntify-eslint/node_modules/eslint/lib/config.js
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/gruntify-eslint/node_modules/eslint/lib/cli-engine.js
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/gruntify-eslint/node_modules/eslint/lib/api.js
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/gruntify-eslint/tasks/eslint.js
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/grunt/lib/grunt/task.js
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/grunt/lib/grunt.js
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/grunt/node_modules/grunt-cli/bin/grunt
- /Users/rua/_automattic/ventures-repos/woocommerce/node_modules/grunt/bin/grunt Use --force to continue.

Aborted due to warnings.
```

Currently I work around this by commenting out the `eslint` task in `Gruntfile.js`. But I'm guessing I'm missing a setup step, or need to install something globally on my dev machine.

**Expected behavior**
- Clone repo.
- Install dependencies e.g. `npm install && composer install`.
- Build with `npm run build` succeeds.

+1 I am also experiencing this issue when running `npm run build:dev` 

Result:
```
Running "eslint:src" (eslint) task
Warning: Cannot read config file: /app/wp-content/plugins/woocommerce/.eslintrc.js
Error: Cannot find module '@automattic/puppeteer-utils'
```

Thanks @roykho So far it's a no go. unsure if it's something to do with the `lando` development environment. trying with WSL ubuntu too but it seems to hang and move painfully slowly through some `reify` things before ultimately failing here... which looks like it's a bit later in the process than before.


```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @automattic/puppeteer-utils@0.0.1 postinstall: `npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @automattic/puppeteer-utils@0.0.1 postinstall script.
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -1,8 +1,6 @@
 /** @format */
 
-const { useE2EEsLintConfig } = require( './tests/e2e/env/config/use-config' );
-
-module.exports = useE2EEsLintConfig( {
+module.exports = {
 	root: true,
 	env: {
 		browser: true,
@@ -30,4 +28,4 @@ module.exports = useE2EEsLintConfig( {
 			jsx: true
 		}
 	},
-} );
+};

